src/store/slices/diagram.ts

```diff
@@ -7,6 +7,8 @@ interface IInitialState {
   saveConfirmationDialog: { isVisible: boolean; data?: any };
   processInfoDialog: { isVisible: boolean; data?: any };
   showDataChannelDialog: { isVisible: boolean; data?: any };
+  deleteDialog: { isVisible: boolean; data?: any };
```

Comment:
Você sabe dizer se o atributo data é sempre um objeto? Se for pode usar o type Record<string, unknow>